README.md

```diff
@@ -98,7 +98,8 @@ There’s also a <a href="https://vimeo.com/168648012">fantastic video</a> on ho
 ## Quick start
 
 1. Clone this repo using `git clone --depth=1 https://github.com/mxstbr/react-boilerplate.git`
-1. Run `npm run setup` to install dependencies and clean the git repo.<br />
+1. We recommend to use yarn for dependency management, do a `yarn install` before doing `npm run setup-yarn`. Check out <a href="https://yarnpkg.com/en/docs/install" target="_blank">here</a> how to install yarn on your OS.
```

Comment:
Hi @sedubois yes you can, but its more obvious for beginners of what it does with npm syntax don't you think so?